dask/utils.py

```diff
@@ -702,7 +703,8 @@ def infer_storage_options(urlpath, inherit_storage_options=None):
     "host": "node", "port": 123, "path": "/mnt/datasets/test.csv",
     "url_query": "q=1", "extra": "value"}
     """
-    if ':\\' in urlpath:
+    # Handle Windows paths including disk name in this special case
+    if re.match(r'[a-zA-Z]:\\', urlpath):
```

Comment:
Yes, they are.

> Because mounted drives are not subject to the 26-drive-letter limit for local drives and mapped network connections, use mounted drives when you want to gain access to more than 26 drives on your computer.

https://support.microsoft.com/en-us/kb/307889